lib/child_process.js

```diff
@@ -347,7 +347,11 @@ function normalizeSpawnArguments(file /*, args, options*/) {
     }
   }
 
-  args.unshift(file);
+  if (typeof options.argv0 === 'string') {
```

Comment:
@jasnell `options.argv0` comes from the caller so it can be anything, unlike `process.argv0` :)